json-endpoints/jetpack/class.jetpack-json-api-plugins-modify-endpoint.php

```diff
@@ -89,7 +89,7 @@
 			'$plugin' => '(string) The plugin ID',
 		),
 		'query_parameters'     => array(
-			'invite_accepted' => '(bool=false) If the user is being created in the invite context',
+			'autoupdate' => '(bool=false) If the user is being created in the invite context',
```

Comment:
Shouldn't the string be updated as well?

Reply:
Good catch!